Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -10675,6 +10686,8 @@
 		44CE2BBE2643799E007AE110 /* WebTextIndicatorLayer.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = WebTextIndicatorLayer.h; sourceTree = "<group>"; };
 		44CF5097299B90600038D284 /* RequestPriority.idl */ = {isa = PBXFileReference; lastKnownFileType = text; path = RequestPriority.idl; sourceTree = "<group>"; };
 		44CF5098299B906F0038D284 /* RequestPriority.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = RequestPriority.h; sourceTree = "<group>"; };
+		44D286EB2CAEB9A000E02E84 /* JSCaretPosition.cpp */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.cpp.cpp; name = JSCaretPosition.cpp; path = "/Users/rbuis/Library/Developer/Library/Developer/Xcode/DerivedData/WebKit-dvwybpzhkzbzyseogofxedcfxxir/Build/Products/Debug/DerivedSources/WebCore/JSCaretPosition.cpp"; sourceTree = "<absolute>"; };
+		44D286ED2CAEB9BA00E02E84 /* JSCaretPosition.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; name = JSCaretPosition.h; path = "/Users/rbuis/Library/Developer/Library/Developer/Xcode/DerivedData/WebKit-dvwybpzhkzbzyseogofxedcfxxir/Build/Products/Debug/DerivedSources/WebCore/JSCaretPosition.h"; sourceTree = "<absolute>"; };
```

Comment:
Can’t check this in with paths that are absolute and include your user name! Needs to be changed to be group-relative.

Reply:
Fixed.